fl/src/federation/client.py

```diff
@@ -1,10 +1,12 @@
 import torch
+from torch.nn.functional import mse_loss
```

Comment:
why do we have client.py in both under federation and under train? is it not a bit confusing?

Reply:
Probably yes. I can't think of a different name for federation/client.py and train/client.py but I agree that it needs to be renamed